### ISSUE TITLE
When opening the sploit_installer through the 2.0.0 homebrew CIA It causes a crash.

### ISSUE THREAD

**Issue by foodcake** [yellows8/oot3dhax#140]
I'm on Luma3DS 11.6.0-39 U, Bottom screen states "Current status: Failed to select the exploit
Error code : 00000002 The title this sploit_installer is running under is not supported."
Top screen says "Initializing... You may press START at any time to return to menu.
Looks like something went wrong."

**Comment by urherenow** [comment]
Are you trolling right now or what? Not only is this not made for 11.6 but if you're using Rosalina, you're on b9s or fastboot3DS, making this 100% obsolete.

**Comment by foodcake** [comment]
No, I am trying to install it onto my cart so I can boot into homebrew with my N3DS 11.4.0-37 U non-CFWed console.

**Comment by hiimzech** [comment]
you need reisix to run the sploits. but you'll run into another issue now where you can't get the payload. I'm looking for a way to update the payload in the save file manually
